tests/unit/test_milvus_backend.py

```
@@ -0,0 +1,258 @@
+
+    def create_namespace(namespace_id: str) -> Namespace:
+        return Namespace(id=namespace_id, created_at=datetime.datetime.now(datetime.UTC))
+
+    manager = MagicMock()
+    manager.__enter__ = Mock(return_value=manager)
+    manager.__exit__ = Mock(return_value=False)
+    manager.create_namespace = create_namespace
+    return manager
+
+
+def always_has_collection(collection_name: str):
```

Ruff (0.14.14)

Unused function argument: `collection_name`

(ARG001)